test/services/publisher_unverified_calculator_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'test_helper'
+
+class PublisherUnverifiedCalculatorTest < ActiveJob::TestCase
+  test "only returns unverified publishers who don't share an email, brave_publisher_id, or phone number with a verified publisher" do
+    # clear database
+    Publisher.delete_all
```

Comment:
i think this is breaking the tests, because the fixtures are deleted and not recreated, impacting subsequent tests

can we comment this out for now?

Reply:
good catch.  i'll adjusts the tests so fixtures do not interfere.